src/service.ui/index.js

```diff
@@ -90,277 +90,179 @@ export const cva = ({ base = "", variants = {}, compoundVariants = [], defaultVa
     defaultVariants,
   });
 
-export default class UIService {
-  isMac = false;
-  isPWA = false;
-  isIOS = false;
-  isAndroid = false;
-  isMobileApp = false;
-  PX_IN_REM = 16;
-  HYPHEN_SYMBOL = "-";
-
-  constructor() {
-    // Needed to avoid error in server context for ssr
-    // TODO: Find way call it in browser
-    // Possible solution: https://github.com/nuxt/nuxt/discussions/7878
-    const isBrowser = typeof window !== "undefined";
-
-    this.isMac = isBrowser && this.checkIsMac();
-    this.isPWA = isBrowser && this.checkIsPWA();
-    this.isIOS = isBrowser && this.checkIsIOS();
-    this.isAndroid = isBrowser && this.checkIsAndroid();
-    this.isMobileApp = this.isPWA || this.isIOS || this.isAndroid;
-
-    this.initTheme();
-  }
+const PX_IN_REM = 16;
+const HYPHEN_SYMBOL = "-";
 
-  initTheme() {
-    const prefersColorSchemeDark = window && window.matchMedia("(prefers-color-scheme: dark)");
+initTheme();
 
-    this.setTheme({ systemDarkMode: prefersColorSchemeDark.matches });
+function initTheme() {
```

Comment:
I think IIEF is good here by the way, please add it again but put inside unnamed arrow function